examples/models/object_detection/faster_rcnn/train.py

```diff
@@ -239,22 +257,6 @@ def pad_fn(examples):
 
 
 with strategy.scope():
-    # The keras huber loss will sum all losses and divide by BS * N * 4, instead we want divide by BS
-    # using NONE reduction as also summing creates issues
-    rpn_reg_loss_fn = tf.keras.losses.Huber(reduction=tf.keras.losses.Reduction.NONE)
-    rpn_cls_loss_fn = tf.keras.losses.BinaryCrossentropy(
-        from_logits=True, reduction=tf.keras.losses.Reduction.NONE
-    )
-    rcnn_reg_loss_fn = tf.keras.losses.Huber(reduction=tf.keras.losses.Reduction.NONE)
-    rcnn_cls_loss_fn = tf.keras.losses.SparseCategoricalCrossentropy(
-        reduction=tf.keras.losses.Reduction.NONE
-    )
-
-    rpn_reg_metric = tf.keras.metrics.Mean()
-    rpn_cls_metric = tf.keras.metrics.Mean()
-    rcnn_reg_metric = tf.keras.metrics.Mean()
-    rcnn_cls_metric = tf.keras.metrics.Mean()
-
     lr_decay = tf.keras.optimizers.schedules.PiecewiseConstantDecay(
         boundaries=[12000 * 16 / global_batch, 16000 * 16 / global_batch],
```

Comment:
i think the boundary here really depends on the size of the dataset (since the ds is not repeat and there isn't a steps per epoch in fit()). We might want to change 12000 and 16000 to something more meaningful (I assume there are 1000 steps per dataset, and 12000 refers to 12 epochs.)